server/routes/index.ts

```diff
@@ -72,10 +58,32 @@ export default function routes(service: Services): Router {
     }
 
     const report = await incidentReportingApi.getReportWithDetailsById(id)
+    const reportedBy = (await manageUsersApiClient.getNamedUser(systemToken, report.reportedBy))?.name
     const prisonerNumbers = report.prisonersInvolved.map(pi => pi.prisonerNumber)
     const prisonersLookup = await offenderSearchApi.getPrisoners(prisonerNumbers)
 
-    res.render('pages/report', { report, prisonersLookup })
+    res.render('pages/report', { report, prisonersLookup, reportedBy })
+  })
+
+  get('/incidents', async (req, res, next) => {
+    const { user } = res.locals
+    const systemToken = await hmppsAuthClient.getSystemClientToken(user.username)
+    const incidentReportingApi = new IncidentReportingApi(systemToken)
+    const manageUsersApiClient = new ManageUsersApiClient()
+
+    const incidents = (
+      await incidentReportingApi.getEvents({
+        // prisonId: user.activeCaseLoadId,
+        // eventDateFrom: new Date('2024-07-30'),
+        // eventDateUntil: new Date('2024-07-30'),
+        // sort: ['eventDateAndTime,ASC'],
+      })
+    )?.content
```

Comment:
I reckon the template should get the whole thing - not just content/the list of incidents - as they're needed to show pagination.

Reply:
made some adjustments to keep the whole response

Reply:
🤔 I think eventually we'd need pagination to work as when we'll have many incidents this will only show the first page (and there will be no way to show the subsequent pages).

But maybe this could go in without that for now? And maybe added separately - @ushkarev what do you reckon?

Reply:
it doesn't have to be done in this pr, but i think some basic filtering and pagination are worth adding. since we would need to deal with that somehow in future anyway and there's nothing more concrete to build 🤷